heft/heft-webpack-basic-tutorial/tsconfig.json

```diff
@@ -3,7 +3,7 @@
 
   "compilerOptions": {
     "outDir": "lib",
-    "rootDir": "src",
+    "rootDirs": ["src/", "temp/sass-ts/"],
```

Comment:
Maybe @iclanton  will remember for certain, but I vaguely remember there being something counterintuitive about the "rootDirs" field that required us to keep both "rootDir" and "rootDirs". I think "rootDir" is used for calculating the root path and if you don't have a TS file in your "src" folder, then you might lose your folder structure without it being also specified.

Reply:
@halfnibble good catch!

You are right, despite the name, `rootDir` and `rootDirs` are very different settings:
- `rootDir` explicitly specifies the folder that will be used as the basis for `lib` output; otherwise it will be inferred from the common parent of the `files`, `include`, etc
- `rootDirs` specifies directories with additional typings that will be overlayed on the main `src` directory tree, under the assumption that those `.js` files will get copied into the same output folder.

Specifying `rootDirs` does not influence the `rootDir`.  

As an experiment, with `"rootDir": "src"` deleted in this branch, if I move all my files to be `src/x/*.ts` they wrongly get compiled to `lib/*.ts` instead of `lib/x/*.ts`.

The naming of these settings is very misleading.